src/static/media-gallery.tsx

```diff
@@ -0,0 +1,38 @@
+export const ACCEPTED_FILE_TYPES = {
+  "image/*": [".jpg", ".jpeg", ".png"],
```

Comment:
also add .JPG, .JPEG and .PNG.  i.e the capital letter extensions as well

Reply:
extensions added .